packages/suite-desktop-core/e2e/support/pageActions/settingsActions.ts

```diff
@@ -1,85 +1,98 @@
-import { Page } from '@playwright/test';
+import { expect, Locator, Page } from '@playwright/test';
 
 import { BackendType, NetworkSymbol } from '@suite-common/wallet-config';
 
-import { waitForDataTestSelector } from '../common';
-
-const settingSectionsLocators = {
-    debug: '@settings/debug/github',
-    general: '@general-settings/language',
-    device: '@settings/device/backup-recovery-seed',
-    wallet: '@settings/wallet/network/btc',
-} as const;
-type Section = keyof typeof settingSectionsLocators;
-
 export class SettingsActions {
     private readonly window: Page;
+    private readonly TIMES_CLICK_TO_SET_DEBUG_MODE = 5;
+    readonly settingsHeader: Locator;
+    readonly debugTabButton: Locator;
+    readonly applicationTabButton: Locator;
+    readonly deviceTabButton: Locator;
+    readonly coinsTabButton: Locator;
+    readonly earlyAccessJoinButton: Locator;
+    readonly earlyAccessConfirmCheck: Locator;
+    readonly earlyAccessConfirmButton: Locator;
+    readonly earlyAccessSkipButton: Locator;
+    readonly settingsCloseButton: Locator;
+    readonly modal: Locator;
+    //coin Advance settings
```

Comment:
I had quite a pickle with naming of these. Originally the naming was refering to networks. I kept that somewhere but change a lot to coins. Thanks to small discussion with Boso.
Please feel free to suggest alternatives. As I have still very limited knowledge of the product and its naming.